src/utils/create-big-number.js

```diff
@@ -1,10 +1,11 @@
 import logError from "utils/log-error";
 import { BigNumber } from "bignumber.js";
 
-export const createBigNumber = (...args) => {
+export const createBigNumber = (value, ...args) => {
   let newBigNumber;
   try {
-    newBigNumber = new BigNumber(...args);
+    const bn = typeof value === "number" ? value.toString() : value;
```

Comment:
Should it ever be anything but a string? If not, try this?
```suggestion
    const bn = `${value}`;
```

Reply:
good point, will update